### ISSUE TITLE
Web admin: Title translation broken ?

### ISSUE THREAD

**Issue by lzammit** [inverse-inc/packetfence#5218]
````
[root@cluster1 pf]# rpm -qa packetfence
packetfence-9.3.9-20200323153530.128892590.0007.devel.el7.x86_64
````

![image](https://user-images.githubusercontent.com/5261214/77348810-7a7ae480-6d10-11ea-91a9-3b86586cee9b.png)
![image](https://user-images.githubusercontent.com/5261214/77348992-bc0b8f80-6d10-11ea-92bb-2bb8dae3d562.png)

**Comment by julsemaan** [comment]
Its not only the translation, its actually the template that doesn't use template toolkit anymore.

@fdurand, I guess we can use golang templates for this, it must not be rocket science but the main question is whether or not the translations are available in golang

**Comment by fdurand** [comment]
Yes i noticed that too, btw it looks to be only for the title.

**Comment by julsemaan** [comment]
We could always just call the page "PacketFence" which doesn't need to be translated

**Comment by nqb** [comment]
For information, installation of devel packages have failed this night on Debian and CentOS after trying to access web admin (just after disabling configurator).

For [CentOS](https://gitlab.com/inverse-inc/packetfence/-/jobs/482763815), it's clearly the issue reported here. 

For [Debian](https://gitlab.com/inverse-inc/packetfence/-/jobs/482763815), issue seems different. @fdurand, could you check why ?

**Comment by cgx** [comment]
Fixed by 1a61033.
